src/App.java

```diff
@@ -1,31 +1,270 @@
 import java.util.Scanner;
 
 public class App {
-    
+
+    public String GREEN = "\033[0;32m"; // GREEN
+    public String BLUE = "\033[0;34m"; // BLUE
+    public String RESET = "\033[0m"; // Text Reset
+    public String RED = "\033[0;31m"; // RED
+    public String PURPLE = "\033[0;35m"; // PURPLE
+
+    /*
+     * The colors are ANSI escape codes that are used to change the color of the
+     * text.
+     * \033[ is the escape code, so all of the colors have it.
+     * - Sometime it is written as ESC or \e[
+     * The part after that is the color code.
+     * - 0;32m is the color code for green
+     * - 0;34m is the color code for blue
+     * - 0;31m is the color code for red
+     * - 0;35m is the color code for purple
+     * - 0m is the color code for reset
+     * learned from
+     * https://stackoverflow.com/questions/5762491/how-to-print-color-in-console-
+     * using-system-out-println
+     */
+
+    // Borders and symbols https://www.w3.org/TR/xml-entity-names/025.html, and
+    // https://www.alt-codes.net/arrow_alt_codes.php
+
+    public void color(String color, String text) {
+        System.out.println(color + text + RESET);
+    }
+
     /**
      * This method is the primary driver method for your program.
      * It should primarily call other methods to do its work.
      * Do NOT edit this method until after you have fully satisfied the reqs for a 5
      */
 
     public void start() {
+        printInstructions();
         String userName = getUserName();
         printGreeting(userName);
-        printInstructions();
         doMadlib();
-        System.out.println("hi");
     }
-    
-    /* Here you should create all missing methods used by the start() method
+
+    /*
+     * Here you should create all missing methods used by the start() method
      * I have created the signature for ONE of the methods below.
      * You can create any other methods you need as well!
-     */ 
-    
+     */
+
+    public void doMadlib() {
+        Scanner input = new Scanner(System.in);
+        System.out.println(BLUE + "│ " + RED + "Do you want to play version 1, 2, or 3?" + RESET);
+        System.out.print(BLUE + "│ " + RESET);
+        String gameMode = input.nextLine();
+        System.out.println(BLUE + "│ " + RESET + "You selected: " + gameMode);
+        System.out.println(BLUE + "├────────────────────────────────────────" + RESET);
+
+        if (gameMode.equals("1")) {
+            System.out.print(BLUE + "│ " + RESET + "Enter a celebrity: \n" + BLUE + "│ ↪ " + RESET);
```

Comment:
It might have simplified the abstraction of your code to make a separate method called `madlibOne()` or something to better organize this code and make it easier to read